docs/how-to/use-flask-extension.rst

```diff
@@ -19,17 +19,21 @@ Example:
     extensions:
       - flask
 
+    flask/install-app:
+      prime:
+        - -srv/flask/app/.git
+        - -srv/flask/app/.venv
+        - -srv/flask/app/.yarn
+        - -srv/flask/app/node_modules
+
 Managing project files with the flask extension
 -----------------------------------------------
 
-By default, all files within the Flask project directory are copied, excluding
-certain common files and directories, such as ``node_modules``. However,
-this behaviour can be tailored to either specifically include or exclude files
-from the Flask project directory in the ROCK image.
+The prime declaration must be included in the specially-named
+``flask/install-app`` section to instruct the flask extension on which files
+to include or exclude from the project directory in the ROCK image.
```

Comment:
I think we should now document the fact that every item in the `prime` list must be prefixed with `srv/flask/app`. This also gives us a chance to talk about this prefix, heh.

Here's a rough first attempt:

```
The extension places the files from the project folder in the `/srv/flask/app` directory in the final image.
Therefore, all inclusions and exclusions must be prefixed with `src/flask/app`.

Reply:
Yes indeed, added, thanks!